pkg/apis/crd/v1alpha2/types.go

```diff
@@ -252,6 +251,9 @@ type ExternalIPPool struct {
 
 	// Specification of the ExternalIPPool.
 	Spec ExternalIPPoolSpec `json:"spec"`
+
+	// The current status of the ExternalIPPool.
+	Status ExternalIPPoolStatus `json:"status"`
```

Comment:
@tnqn we don't need `omitempty` for this? I know that when kubebuilder generates CRD types, it always adds the `omitempty` tag for the `Status` field.

I also see it for upstream resources (at least for `Pod`)

Reply:
It seems that there is no `omitempty` tag for all the `Status` field in the project. I think we can take another PR to implement it? @antoninbas @tnqn

Reply:
Thanks for confirming guys.